src/vaishnaviVJun20_arrayExamples/ReverseInteger.java

```diff
@@ -0,0 +1,21 @@
+package vaishnaviVJun20_arrayExamples;
+
+//WAP to revere number (Consider Input as Integer not as String)
+
+public class ReverseInteger {
+	
+	void reverseIntegerNumber(int num) {
+		System.out.print("Reverse number is:");
+		while(num>0) {
+			int reverse = num%10;
+			num = num/10;
+			System.out.print(reverse);
```

Comment:
Here you are directly printing the reverse order of the given number, 
Implement integer reverse logic - which is addition of one more statement

Reply:
--suggested changes done